packages/core-data/src/entities.js

```diff
@@ -196,8 +166,16 @@ function* loadPostTypeEntities() {
 				selectionEnd: true,
 			},
 			mergedEdits: { meta: true },
-			getTitle: getPostTypeTitle( name ),
-			__unstablePrePersist: getPostTypePrePersistHandler( name ),
+			getTitle: isTemplate
+				? ( record ) =>
```

Comment:
Nit pick: You could use one function and check for `isTemplate` in the last condition, since the other two are identical.